dolphinscheduler-registry/dolphinscheduler-registry-api/src/main/java/org/apache/dolphinscheduler/registry/api/ha/AbstractServerStatusChangeListener.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.registry.api.ha;
+
+public abstract class AbstractServerStatusChangeListener implements ServerStatusChangeListener {
+
+    @Override
+    public void change(HAServer.ServerStatus originStatus, HAServer.ServerStatus currentStatus) {
+        if (originStatus == HAServer.ServerStatus.ACTIVE) {
+            if (currentStatus == HAServer.ServerStatus.STAND_BY) {
+                changeToStandBy();
+            } else {
+                throw new IllegalStateException("The status change from ACTIVE to ACTIVE is not allowed.");
+            }
+        } else if (originStatus == HAServer.ServerStatus.STAND_BY) {
+            if (currentStatus == HAServer.ServerStatus.ACTIVE) {
+                changeToActive();
+            } else {
+                throw new IllegalStateException("The status change from STAND_BY to STAND_BY is not allowed.");
```

Comment:
If one alert server's original status is `Standby`, and it doesn't acquire lock successfully in new election, I don't think we should throw exception here.

Reply:
Agree with you, I removed this logic.